plot_continuous_monitor.py

```diff
@@ -87,9 +87,9 @@ def plot_continuous_monitor(filename, open_graphs=False):
       running_disk_monotasks_info = json_data["Running Disk Monotasks"]
       RUNNING_MONOTASKS_KEY = "Running And Queued Monotasks"
       if (len(running_disk_monotasks_info) > 0):
-        xvdf_running_disk_monotasks = running_disk_monotasks_info[0][RUNNING_MONOTASKS_KEY]
+        xvdb_running_disk_monotasks = running_disk_monotasks_info[0][RUNNING_MONOTASKS_KEY]
```

Comment:
Can we directly read the disk ID to decide this, to make this code somewhat more robust?